src/main/java/repository/InteractionWithDB.java

```diff
@@ -0,0 +1,116 @@
+package repository;
+
+import model.CarEntity;
+
+import java.sql.*;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class InteractionWithDB {
```

Comment:
zmien nazwę na dedykowaną dla samochodów, np CarsRepository. Jak dodasz jeszcze właścicieli aut do bazy to nie nazwiesz przecież tamtej klasy InteractionWithDB2 📦